go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/ZupIT/horusec
 go 1.17
 
 require (
-	github.com/ZupIT/horusec-devkit v1.0.22
+	github.com/ZupIT/horusec-devkit v1.0.23-0.20220221194704-5b84b7847e09
```

Comment:
A new devkit release should be released before merge

Reply:
done